generators/server/templates/pom.xml.ejs

```diff
@@ -97,9 +97,13 @@
         <!-- The hibernate version should match the one managed by
         https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/${spring-boot.version} -->
         <hibernate.version><%= HIBERNATE_VERSION %></hibernate.version>
+<%_ } _%>
+<%_ if (enableLiquibase) { _%>
         <!-- The liquibase version should match the one managed by
         https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/${spring-boot.version} -->
         <liquibase.version><%- javaDependencies.liquibase %></liquibase.version>
+<%_ } _%>
+<%_ if (databaseTypeSql) { _%>
```

Comment:
The condition should be included in the enableliquibase, no?

Reply:
```suggestion
<%_ if (enableLiquibase && databaseTypeSql) { _%>
```